docs/en/compute/Google-Dataproc.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: global
+title: Running Alluxio on Google Cloud Dataproc
+nickname: Google Dataproc
+group: Data Applications
+priority: 0
+---
+
+This guide describes how to configure Alluxio to run on [Google Cloud Dataproc](https://cloud.google.com/dataproc).
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+Google Cloud Dataproc is a managed on-demand service to run Spark and Hadoop compute workloads.
+It manages the deployment of various Hadoop Services and allows for hooks into these services for
+customizations.
+Aside from the added performance benefits of caching, Alluxio also enables users to run compute 
+workloads against on-premise storage or even a different cloud provider's storage i.e. AWS, Azure
+Blob Store.
+
+## Prerequisites
+
+* Account with Cloud Dataproc API enabled
+* A GCS Bucket
+* gcloud CLI: Make sure that the AWS CLI is set up and ready with the required AWS Access/Secret key
+
+An GCS bucket can be configured as Alluxio's Root Under File System and to serve as the location for the
+bootstrap script.
+If required, the root UFS can be reconfigured to be HDFS or any other supported under store.
+
+## Basic Setup
+
+When creating a Dataproc cluster, Alluxio can be installed using an
+[initialization action](https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/init-actions)
+
+1. The Alluxio initialization action is hosted in a publicly readable
+GCS location **gs://alluxio-public/alluxio.sh**.
```

Comment:
We should standardize the format as "gs://alluxio-public/dataproc/2.0.1/alluxio-dataproc.sh" @Xenorith please weigh in on this.

Reply:
👍 though said bucket has yet to be created

Reply:
i lied. it already exists and has a script in the exact location as described...